.github/workflows/test-pull-request.yaml

```diff
@@ -26,10 +26,34 @@ jobs:
         with:
           node-version: "16.x"
 
-      # install deps for frontend and library and build both
-      - name: Install library and frontend deps and build
+      # install deps for Library development and build
+      - name: Install library for development
+        working-directory: ./library
+        id: npm
+        run: |
+          npm ci
+          npm run build-dev
+
+      # install deps for Library production and build
+      - name: Install library for production
+        working-directory: ./library
+        id: npm
```

Comment:
Remove the id here we dont need it. Having the same id for multiple steps is causing the action to fail, see https://github.com/ScalablePixelStreaming/Frontend/actions/runs/4559049649

Reply:
resolved in 57089f65c30a4d20bdbe5aae074acd237aa69b81